packages/mongo/oplog_tests.js

```diff
@@ -180,6 +182,116 @@ process.env.MONGO_OPLOG_URL && Tinytest.addAsync(
   },
 );
 
+let defaultOplogHandle = MongoInternals.defaultRemoteCollectionDriver().mongo._oplogHandle;
+
+async function oplogOptionsTest({
+  test,
+  includeCollectionName,
+  excludeCollectionName,
+  mongoPackageSettings = {}
+}) {
+  if (!Meteor.settings.packages) Meteor.settings.packages = {};
+  if (!Meteor.settings.packages.mongo) Meteor.settings.packages.mongo = mongoPackageSettings;
+
+  const myOplogHandle = new MongoInternals.OplogHandle(process.env.MONGO_OPLOG_URL, 'meteor');
+  MongoInternals.defaultRemoteCollectionDriver().mongo._setOplogHandle(myOplogHandle);
```

Comment:
Same goes for the `OplogHandle`.

Reply:
I can only reset the used `OplogHandle` like so, but I don't think there is any way to "destroy" `myOplogHandle`, is there?!

Reset:
```js
MongoInternals.defaultRemoteCollectionDriver().mongo._setOplogHandle(defaultOplogHandle);
```

Reply:
My point is that you should extract the previous handle before setting the new one and set the old one back after the test.

Reply:
That is exactly what I am doing!

Reply:
No, you're extracting it on the top level, not in the test. That's a significant difference, as the top-level code executes first only to register the tests.

_(Also, I don't know how to understand your exclamation marks... I hope you're excited and not getting angry at me.)_